src/block-components/button/attributes.js

```diff
@@ -0,0 +1,51 @@
+import { addLinkAttributes } from '../helpers/link'
+import { addBorderAttributes } from '../helpers/borders'
+
+export const addAttributes = ( attrObject, options = {} ) => {
+	const {
+		selector = '.stk-button__button',
+	} = options
+	attrObject.add( {
+		attributes: {
+			buttonPadding: {
+				stkResponsive: true,
+				type: 'object',
```

Comment:
you can use `stkUnits: 'px'` here instead of defining your own `buttonPaddingUnit` below